api/models.py

```diff
@@ -91,6 +91,9 @@ def update(self):
     def create_indexes(cls, collection):
         """Method to create indexes"""
 
+    def set_timeout(self, hours: int = 24):
```

Comment:
We don't really use type annotations in any other place, I guess we can get rid of it in this method, too. Having said that it'd be good to refactor api code to use type annotations, but that's something that needs to be planned.

I think we should not have default value in the base method. It looks a bit odd to have a method without implementation, but with a default value for the parameter. `hours` parameter should stay there, but without default value

Reply:
I have added a static value of hours in `api.db` as a workaround to resolve the current failure.
This will be changed later as @gctucker  is already working on an implementation for dynamic timeout and holdoff.